src/seedu/addressbook/Main.java

```diff
@@ -107,6 +108,7 @@ private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
             CommandResult result = command.execute();
+            storage.fileExist();
```

Comment:
This may be a too high-level method to check for this low-level problem. `StorageFile::save()` may be a more suitable place to detect it. However, this is a suitable place to handle the exception thrown from `StorageFile::save()`